prime_no.rb

```diff
@@ -0,0 +1,16 @@
+puts "Enter number:"
+
+n = gets.chomp.to_i
+
+if (n < 2)
+    puts "No. is not prime"
+else
+    (2..n-1).each do |ele|
+        if(n % ele) == 0
+            puts "No. is not prime"
+        else
+            puts "No. is prime"
```

Comment:
This logic will not work correctly, because until loop for every number  "No is not prime" or "No is prime" will get printed. which is incorrect
If number is divisible by any of the number then no need to check until loop. directly you can break the loop and print that "No is not prime".
Also you can use flag for continue or break of loop